Cakefile

```diff
@@ -35,6 +35,11 @@ build = (callback) ->
             console.error err
         else
             build_webpack () ->
+                jsPath = build_dir + '/APIBlueprintImporter.js'
+                js = fs.readFileSync(jsPath, 'utf-8')
+                js = js.replace('var Module={"ready"', 'var Module={"ENVIRONMENT":"WEB","ready"')
```

Comment:
I've tried various combinations of using the webpack `DefinePlugin`.

```diff
diff --git a/webpack.config.babel.js b/webpack.config.babel.js
index e012874..92f0272 100644
--- a/webpack.config.babel.js
+++ b/webpack.config.babel.js
@@ -23,5 +23,10 @@ module.exports = {
         exclude: /node_modules/,
       }
     ]
-  }
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'Module': JSON.stringify({'ENVIRONMENT': 'WEB'})
+    })
+  ]
 }
```

It would seem that the webpack DefinePlugin is not placing `Module` inside the resultant javascript file at all.